doc/src/guides/solving/solve-ode.md

```diff
@@ -0,0 +1,262 @@
+
+```py
+>>> from sympy import symbols, Eq, Function, dsolve
+>>> y, z = symbols("y z", cls=Function)
+>>> x = symbols("x")
+>>> eqs = [Eq(y(x).diff(x)**2, z(x)**2), Eq(z(x).diff(x), z(x))]
+>>> solutions = dsolve(eqs); solutions
+[[Eq(y(x), C1 - C2*exp(x)), Eq(z(x), C2*exp(x))], [Eq(y(x), C1 + C2*exp(x)), Eq(z(x), C2*exp(x))]]
+>>> solutions_list = [] # nested list approach
+>>> for solution in solutions:
+...     solution_dict = {}
+...     for fn in solution:
+...             solution_dict.update({fn.lhs: fn.rhs})
+...     solutions_list.append(solution_dict)
+>>> solutions_list
+[{y(x): C1 - C2*exp(x), z(x): C2*exp(x)}, {y(x): C1 + C2*exp(x), z(x): C2*exp(x)}]
+>>> solutions_list = [{fn.lhs:fn.rhs for fn in solution} for solution in solutions]
+>>> solutions_list # nested comprehension approach
+[{y(x): C1 - C2*exp(x), z(x): C2*exp(x)}, {y(x): C1 + C2*exp(x), z(x): C2*exp(x)}]
+>>> solutions_list[0][y(x)]
+C1 - C2*exp(x)
+```
+
+### Work With Arbitrary Constants *include such a section?*
+
+## Ordinary Differential Equation Solving Hints
+
+### Return Unevaluated Integrals
+
+By default, {func}`~.dsolve` attempts to evaluate the integrals it produces to
+solve your ordinary differential equation. You can disable evaluation of the
+integrals by using {ref}`hints` ending with `_Integral`, for example
+`separable_Integral`. This is useful because
+{func}`~sympy.core.expr.Expr.integrate` is an expensive routine. SymPy may hang
+(appear to never complete the operation) because of a difficult or impossible
+integral, so using an `_Integral` hint will at least return an (unintegrated)
+result, which you can then consider. The simplest way to disable integration is
+with the `all_Integral` hint because you do not need to know which hint to
+supply: for any hint with a corresponding `_Integral` hint, `all_Integral` only
+returns the `_Integral` hint.
+
+### Select a Specific Solver
+
+You may wish to select a specific solver using a hint for a couple of reasons:
+- educational purposes: for example if you are learning about a specific method
+  to solve ODEs and want to get a result that exactly matches that method
+- form of the result: sometimes an ODE can be solved by many different solvers,
+  and they can return different results. They will be mathematically equivalent,
+  though the arbitrary constants may not be. {func}`~.dsolve` by default tries
+  to use the "best" solvers first, which are most likely to return the most
+  usable output, but it is not a perfect heuristic. For example, the "best"
+  solver may produce a result with an integral that SymPy cannot solve, but
+  another solver may produce a different integral that SymPy can solve. So if
+  the solution isn't in a form you like, you can try other hints to check
+  whether they give a preferable result.
+
+## Not All Equations Can Be Solved
+
+### Equations With No Solution
+
+*Equations with no solution content*
+
+### Equations With No Analytical Solution
```

Comment:
This looks like a bug, although for a system you should specify the list of functions as the second argument to dsolve (also a good idea for single ODEs, although it will do it automatically if there is only one function in the expression).

Reply:
That's one of the legacy solvers that are mostly just broken and should probably just be removed:
https://github.com/sympy/sympy/wiki/ODE-Systems-roadmap#nonlinear-solvers---eliminating-the-independent-variable

Reply:
Thanks; I added that you supply a list of functions to solve for.

I'm still seeking an example of an ODE (or system of) where there isn't a closed-form solution (and SymPy recognizes that), and would appreciate any ideas!

Reply:
Maybe a double pendulum. See https://scienceworld.wolfram.com/physics/DoublePendulum.html (eqs 14 and 19). Here I simplified them by setting the masses and lengths to 1, although I'm sure it would give the same thing too if I included them:

```py
>>> theta1, theta2 = symbols('theta1 theta2', cls=Function)
>>> g = symbols('g')
>>> eq1 = 2*theta1(t).diff(t, t) + theta2(t).diff(t, t)*cos(theta1(t) - theta2(t)) + theta2(t).diff(t)**2*sin(theta1(t) - theta2(t)) + 2*g*sin(theta1(t))
>>> eq2 = theta2(t).diff(t, t) + theta1(t).diff(t, t)*cos(theta1(t) - theta2(t)) - theta1(t).diff(t)**2*sin(theta1(t) - theta2(t)) + g*sin(theta2(t))
>>> dsolve([eq1, eq2], [theta1(t), theta2(t)])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/aaronmeurer/Documents/Python/sympy/sympy/./sympy/solvers/ode/ode.py", line 589, in dsolve
    raise NotImplementedError
NotImplementedError
```

(please double check that I transcribed it correctly)